data/events.xml

```diff
@@ -1,6 +1,246 @@
+        <subject>Spring Boot</subject>
+        <speaker>Yousuf Haque</speaker>
+        <startDate>2017-02-15</startDate>
+        <endDate>2017-02-15</endDate>
+        <location>New York, USA</location>
+        <lang>en</lang>
+        <description>
+            <![CDATA[
+<p>Kotlin and Spring Boot were both designed to help developers move faster. This talk will demonstrate how the combination of the two technologies can make web service development fast, and easy. We will try to show you how to make CRUD apps using Spring Data, and Google’s Firebase.</p>
+]]>
+        </description>
+    </event>
+    <event>
+        <title>Callista Developer's Conference</title>
+        <url>http://us3.campaign-archive2.com/?u=51fba111f275a6a984e600b9e&id=67163cdacf&e=ced45a4735</url>
+        <subject>Kotlin - ett elegant nytt språk för JVM</subject>
+        <speaker>Jesper Holmberg</speaker>
+        <startDate>2017-01-25</startDate>
+        <endDate>2017-01-25</endDate>
+        <location>Stockholm, Sweden</location>
+        <lang>sv</lang>
+        <description>
+            <![CDATA[
+<p>Kotlin - ett elegant nytt språk för JVM</p>
+]]>
+        </description>
+    </event>
+    <event>
+        <title>January Kotlin Meetup</title>
+        <url>https://www.meetup.com/Kotlin-Vienna/events/236959058/</url>
+        <subject>Show and Tell: Creating a Swing Application with Kotlin</subject>
+        <speaker>Christoph Pickl</speaker>
+        <startDate>2017-01-31</startDate>
+        <endDate>2017-01-31</endDate>
+        <location>Vienna, Austria</location>
+        <lang>en</lang>
+        <description>
+            <![CDATA[
+<p>As a (future) Shiatsu practitioner and IT nerd, I've developed my very own software called Gadsu to manage my clients and their treatments, generating printout protocols for my school, synchronizing appointments with Google Calender and much more. </p>
+]]>
+        </description>
+    </event>
+    <event>
+        <title>January Kotlin Meetup</title>
+        <url>https://www.meetup.com/Kotlin-Vienna/events/236959058/</url>
+        <subject>Spring Boot Microservices with Kotlin</subject>
+        <speaker>Christoph Leiter</speaker>
+        <startDate>2017-01-31</startDate>
+        <endDate>2017-01-31</endDate>
+        <location>Vienna, Austria</location>
+        <lang>en</lang>
+        <description>
+            <![CDATA[
+<p>I spent the last 6 month creating a project with about a dozen microservices which is written 100% in Kotlin. In this talk I will tell you what I've learned and share some tips on how to get the most out of Spring Boot, Kotlin and the ecosystem.</p>
+]]>
+        </description>
+    </event>
+    <event>
+        <lang>de</lang>
+        <startDate>2017-01-31</startDate>
+        <endDate>2017-01-31</endDate>
+        <speaker>Markus Winand</speaker>
+        <location>Graz, Austria</location>
+        <title>Java User Group Graz</title>
+        <subject>Modern SQL</subject>
```

Comment:
Hi! 
Looks like a mistake (probably it was announced with Kotlin mention). I don't find Kotlin mentions there now, so I will remove it.
Thank you!